### PR TITLE
Create Grubstreet.com.xml

### PR DESCRIPTION
**Pull request by ryanbr**

### CODE REVIEW

**Comment by ryanbr** [comment]
Those regional subdomains all redirect or fail on https, maybe redirect? Let me know if I need to remove these or not

File: src/chrome/content/rules/Grubstreet.com.xml [hunk 1]
```diff
@@ -6,13 +6,33 @@
...
 	<target host="www.grubstreet.com" />
 
+	<rule from="^http://boston\.grubstreet\.com/" to="https://www.grubstreet.com/" />
+	<rule from="^http://chicago\.grubstreet\.com/" to="https://www.grubstreet.com/" />
+	<rule from="^http://losangeles\.grubstreet\.com/" to="https://www.grubstreet.com/" />
+	<rule from="^http://newyork\.grubstreet\.com/" to="https://www.grubstreet.com/" />
+	<rule from="^http://philadelphia\.grubstreet\.com/" to="https://www.grubstreet.com/" />
+	<rule from="^http://sanfrancisco\.grubstreet\.com/" to="https://www.grubstreet.com/" />
+
+	<rule from="^http://grubstreet\.com/" to="https://www.grubstreet.com/" />
```
Comment (J0WI) [inline src/chrome/content/rules/Grubstreet.com.xml]: You also need to add them as target

Comment (ryanbr) [reply src/chrome/content/rules/Grubstreet.com.xml]: Actually is needed if they're redirecting them to www  anyway?

Comment (J0WI) [reply src/chrome/content/rules/Grubstreet.com.xml]: You can only redirect `from` `target host`s

File: src/chrome/content/rules/Grubstreet.com.xml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+	Refused:
+		grubstreet.com (non-www)
+		boston.grubstreet.com
+		chicago.grubstreet.com
+		losangeles.grubstreet.com
+		newyork.grubstreet.com
+		philadelphia.grubstreet.com
+		sanfrancisco.grubstreet.com
+-->
+<ruleset name="grubstreet.com">
```
Comment (J0WI) [inline src/chrome/content/rules/Grubstreet.com.xml]:
```suggestion
<ruleset name="grubstreet.com">
	<target host="grubstreet.com">
```

File: src/chrome/content/rules/Grubstreet.com.xml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+	<target host="grubstreet.com">
```
Comment (J0WI) [inline src/chrome/content/rules/Grubstreet.com.xml]:
```suggestion
	<target host="grubstreet.com" />
```

Sorry, my suggestion had a typo

**Pull request merged**